docs/OnnxReleases.md

```diff
@@ -1,59 +1,51 @@
-* Make sure all the git submodules are updated
+* Create a release branch, for ex rel-1.7.0
+* Create a PR against release branch to update VERSION_NUMBER for release verification in TestPypi. Make sure CI is clean.
+* After PR merged, download Windows wheels from git repo Actions, https://github.com/onnx/onnx/actions, and use the commit ID to update .travis in wheel-builder
+* Create a branch pypi-test in onnx/wheel-builder repo and a PR to trigger CI to upload release candidate wheels for Linux and OSX to TestPypi
+* Upload Windows wheels to TestPypi ``python -m twine upload --verbose *.whl --repository-url https://test.pypi.org/legacy/ -u onnx -p pwd``
+* Build ONNX source distribution
     * ``git submodule update –init``
-* Make sure the git checkout is clean –
     * Do ``git clean -nxd`` and make sure that none of the auto-generated header files *like* the following are not present.
         * onnx/onnx-operators.pb.cc
         * onnx/onnx-operator.pb.h
         * onnx/onnx.pb.cc
         * onnx/onnx.pb.h
     * If they are present run ``git clean -ixd`` and remove those files from your local branch
-* Do ``python setup.py sdist`` to generate the source distribution.
-* Put the following content into ``~/.pypirc`` file:
-```
-[distutils]
-index-servers =
-  pypi
-  pypitest
- 
-[pypi]
-username=<username>
-password=<password>
- 
-[pypitest]
-repository=https://test.pypi.org/legacy/
-username=<username>
-password=<password>
-```
-* Do ``twine upload dist/* -r pypitest`` to upload it to the test instance of pypi.
-* After uploading to pypitest, you can test the source distribution by doing ``pip install –index-url https://test.pypi.org/simple/ onnx`` in a new environment. Test this installation with different environments and versions of protobuf binaries.
-* *NOTE - Once a distribution is uploaded to pypi, you cannot overwrite it on the same pypi instance.*
-* Once completely verified do, ``twine upload dist/*``  to upload to the official pypi.
-* *Conda - *
+    * ``python setup.py sdist``
+* Upload source distribution to TestPypi ``python -m twine upload --verbose dist/* -r --repository-url https://test.pypi.org/legacy/ -u onnx -p pwd``
+* After uploading to TestPypi, you can test the source distribution by doing ``pip install –index-url https://test.pypi.org/simple/ onnx`` in a new environment. Test this installation with different environments and versions of protobuf binaries.
+* Verify the release candidate in TestPypi by release test team. The communication is via gitter and an github issue.
+* Prepare a change log for the release
+    * Draft a new release statement - https://github.com/onnx/onnx/releases listing out the new features and bug fixes, and potential changes being introduced in the release.
+* Once the release candidate is completely verified
+  * Create a release in onnx/wheel-builder repo to trigger CI to upload release packages for Linux and OSX to Pypi
```

Comment:
what is the process for this step? In the past PRs I only see the ones to upload to test pypi. I have not seen any PR in wheel builder repo to upload to pypi

Reply:
The official build and upload to pypi is currently triggered by having a new release/tag in wheel-builder. That is the reason we created release v1.7.0 there.